### ISSUE TITLE
Review use of timezones on timestamps

### ISSUE THREAD

**Issue by jonkiparsky** [bethlakshmi/GBE2#597]
It's not clear to me that we have a consistent standard for timezone usage. Looking through the codebase, I'm pretty sure we're using GMT in some places, America/New York in others, and literal Boston time localized to GMT in others. 

We need to determine a standard, and apply it. 
(this ticket should not be taken up until the developers have consensus on what the standard should be and how to go about applying it)

**Comment by jonkiparsky** [comment]
Marking this as low priority, because it's only a minor horror in the ghastly mess we've made.

**Comment by burlexpo** [comment]
This subject came up as a sidebar on Thursday.  I believe the consensus 
was we were using Zulu time (GMT), although I have no idea if we're 
using it throughout.

**Comment by jonkiparsky** [comment]
We're not.

**Comment by bethlakshmi** [comment]
Long ago, I think our thought (UI wise) was to always display EST.

Storage wise, GMT would be nice, but I'm not sure we ever agreed.

**Comment by BenignCremator** [comment]
On 12/07/2015 07:13 PM, bethlakshmi wrote:

> Long ago, I think our thought (UI wise) was to always display EST.

> Storage wise, GMT would be nice, but I'm not sure we ever agreed.

When we first started this project up, we settled upon GMT in the DB and 
pretty much everywhere under the hood, and only translate to a local 
timezone at the Views level.  I do not believe we ever acted on that, 
however.  This is not the first time we have had this conversation.  Not 
the third, either.  Maybe the fourth.

Hunter
